CalendarToSlack/Updater.cs

```diff
@@ -208,21 +216,25 @@ private void MakeSlackApiCalls(RegisteredUser user, Presence presence, string st
             LegacyFreeBusyStatus.Free,
         };
 
-        // Returns null if the user should have no status message.
-        private static string GetUserMessage(CalendarEvent ev, RegisteredUser user)
+        // Returns null if the user's status should not be updated.
+        private static CustomStatus GetCustomStatusForCalendarEvent(CalendarEvent ev, RegisteredUser user)
         {
             if (ev == null)
             {
-                return null;
+                return user.SlackUserInfo.DefaultCustomStatus;
```

Comment:
Should this still be `null`? I'll need to dig through a bit, but will this cause unnecessary calls to the Slack API when the status shouldn't change?

Reply:
The internal logic here is "if there's no event happening, the user should return to their default status." There's also a check that happens outside this method that checks if the _result_ of this method is the same as their current custom status (held in memory). If it's the same, we don't send an update.

Reply:
It turns out we already handle this case [elsewhere](https://github.com/robhruska/CalendarToSlack/blob/master/CalendarToSlack/Updater.cs#L166-L181), so I probably _could_ just return `null` here

Reply:
👍